EEBOS/owl-ui
# sc-889 - Feature - Carousel Component

### Short summary
Add a carousel component to Owl UI using [react bootstrap](https://react-bootstrap.github.io/components/carousel/)

You might notice that the sliding animation acts kind of weird and doesn't match the [React Bootstrap example](https://react-bootstrap.github.io/components/carousel/#example), that happens because of the utility classes that we are not able to prefix. Once we get a solution to that problem this issue will be fixed. 

For now, you can review the code and the elements rendered and then we'll get back to fix the styles before merging them into the main branch.

---

### Test steps

1. Run `yarn`
2. Run `yarn start:storybook` and check out the Carousel component.
